phpunit.xml.dist

```diff
@@ -45,4 +45,10 @@
     <listeners>
         <listener class="Symfony\Bridge\PhpUnit\SymfonyTestsListener" />
     </listeners>
+
+    <groups>
+        <exclude>
+            <group>Broken</group>
```

Comment:
What is this ignoring? For now, only 'testSwitchTranslationSetRoundtrip' seems to be in that group, but I assume that's temporary? Is this group defined just to make us have temporarily inactive tests, or is there something deeper here?

If we need to fix tests, let's document in a todo or make it clear that "Broken" anything is temporary after merge (or decide we only merge when it's not broken?)

Reply:
Good point about marking for follow-up.

My understanding was that this was more about being able to merg the code as-is, for the sake of history (and maybe licensing?) as much as anything, and then we'll get to work fixing it all up.

Reply:
We shouldn't keep unnecessary or broken things in this system. If there's a licensing issue, let's see how to solve it, but we can probably solve it by referring to the old without outright having to copy it.

But yeah, my understanding is that this is done in steps, so the "broken" marker is temporary while it's fixed, which is fine if we make sure it actually happens -- and if not, then we need to remove it from the codebase and see whether we need more tests.

Reply:
https://phabricator.wikimedia.org/T206896

Reply:
Okay, fair enough, but let's try and put those as code comments too. I am always a bit meh when the code doesn't reflect the plans -- especially when the ticket is in Phab and the code is on github. We could add a "TODO: This will be fixed as part of [ticketnumber]" or something in the code, just to make things clear.